components/QrCodeOverlay.tsx

```diff
@@ -37,31 +37,23 @@ export const QrCodeOverlay: React.FC<QrCodeOverlayProps> = props => {
     return qrData;
   }
 
-  const [base64String, setBase64String] = useState('');
-
   let qrRef = useRef(null);
 
-  function updateBase64() {
+  function sharedQRCodeTapped() {
```

Comment:
Since const shareStatus = await shareImageToAllSupportedApps(options);  is async function , not able to use directly inside qrRef scope. Any ways to use that?